FirebaseDataConnect/Sources/Internal/GrpcClient.swift

```diff
@@ -196,22 +194,20 @@ actor GrpcClient: CustomStringConvertible {
     }
 
     // Add AppCheck token if available
-    if appCheckEnabled {
-      do {
-        if let token = try await appCheck?.token(forcingRefresh: false) {
-          headers.add(name: RequestHeaders.appCheckHeader, value: "\(token)")
-          FirebaseLogger.dataConnect
-            .debug("App Check token added: \(token)")
-        } else {
-          FirebaseLogger.dataConnect
-            .debug("App Check token unavailable. Not adding App Check header.")
-        }
-      } catch {
+    do {
+      if let token = try await appCheck?.token(forcingRefresh: false) {
```

Comment:
Should appCheck be accessed via interop like the other products?

Reply:
Thanks for pointing that out. I will merge this public version into the new repo for now and explore interop way.